templates/mod/status.conf.erb

```diff
@@ -1,14 +0,0 @@
-<Location <%= @status_path %>>
-    SetHandler server-status
-    <%-# From Puppet 4.2 up, replace: -%>
-    <%-# "scope.function_template(["apache/mod/<Template>"])" -%>
-    <%-# with: -%>
-    <%-# "scope.call_function('template', ["apache/mod/<Template>"])" -%>
-<%= scope.function_template(["apache/mod/_require.erb"]) -%>
-</Location>
-ExtendedStatus <%= @extended_status %>
-
-<IfModule mod_proxy.c>
-    # Show Proxy LoadBalancer status in mod_status
-    ProxyStatus On
```

Comment:
Probably not critical and perhaps i missed something.
I did not found in epp templates a place where `ProxyStatus` is handled.

Reply:
`ProxyStatus` is handled in the EPP template just like in the ERB one: no direct user control on this setting, when mod_status is enabled, and if mod_proxy is also loaded, the setting is enabled.

In these mod_status files, the conversion to EPP was started but not finished (the template was not used), so I just completed to pass the expected parameters, removed the legacy comments that caused extra space to be output with EPP.  The GitHub diff is cut before the relevant lines, but clicking on `...` you should see the original code :wink: